libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.component.ts

```diff
@@ -81,11 +119,78 @@ export class WattTimepickerComponent extends WattPickerBase {
   @ViewChild('endTimeInput')
   endInput!: ElementRef;
 
+  /**
+   * @ignore
+   */
+  @ViewChild('overlay')
+  overlay!: ElementRef;
+
+  /**
+   * @ignore
+   */
+  sliderId = `${this.id}-slider`;
+
+  /**
+   * Used for defining a relationship between the time picker and
+   * the slider overlay (since the DOM hierarchy cannot be used).
+   * @ignore
+   */
+  @HostBinding('attr.aria-owns')
+  ariaOwns = this.sliderId;
+
   /**
    * @ignore
    */
   protected _placeholder = hoursMinutesPlaceholder;
 
+  /**
+   * Whether the slider is open.
+   * @ignore
+   */
+  sliderOpen = false;
+
+  /**
+   * @ignore
+   */
+  sliderSteps = [...Array(96).keys()].map((x) => x * 15).concat(1439);
+
+  /**
+   * @ignore
+   */
+  sliderChange$ = new BehaviorSubject(initialSliderValue);
+
+  /**
+   * @ignore
+   */
+  get sliderValue() {
```

Comment:
suggestion: add return type

```suggestion
  get sliderValue(): WattSliderValue {
```

Reply:
45f9ee8fa460d32db57ca231df7d5f0e89d93e02